powershell/Tests/L0/Get-Endpoint.WritesErrorIfRequired.ps1

```diff
@@ -14,7 +14,7 @@ Invoke-VstsTaskScript -ScriptBlock {
         $env:ENDPOINT_AUTH_SOMENAME = $variableSet.Auth
 
         # Act.
-        $actual = Get-VstsEndpoint -Name 'SomeName' -Require 2>&1
+        $actual = Get-VstsEndpoint -Name 'SomeName' -Require -ErrorAction Continue 2>&1
```

Comment:
I changed the test infrastructure for the task-lib repo in my last merge. There's some slight differences in the infra from the tasks repo, due to the tests in the tasks repo now leveraging Invoke-VstsTaskScript to run the tests. After I changed the infrastructure, a couple tests were failing, so i'm fixing those as part of this PR.